### PR TITLE
Adding support for fzf-lua in search command

### PR DESCRIPTION
**Pull request by saccarosium**
Currently the only fuzzy finders implemented are telescope.nvim and fzf.vim. 
I use [fzf-lua](https://github.com/ibhagwan/fzf-lua), which is a lua based fzf plugin, that work like telescope but uses fzf in the background for better performance and, in my opinion, better fuzzy algorithm.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ Plug 'hrsh7th/nvim-cmp'
 Plug 'junegunn/fzf', { 'do': { -> fzf#install() } }
 Plug 'junegunn/fzf.vim'
 
+" (optional) for :ObsidianSearch command unless you use telescope:
```
Comment (epwalsh) [inline README.md:54]:
```suggestion
" (optional) another alternative for the :ObsidianSearch command:
```

**Review by epwalsh — COMMENTED** [review]
Didn't know about `fzf-lua`! Thanks for this, just one comment

**Comment by saccarosium** [comment]
Fixed comment and added fix in search.

**Comment by epwalsh** [comment]
Can you just add an entry to the CHANGELOG under `### Added` that says something like "Added support for `fzf-lua`."

**Comment by saccarosium** [comment]
> Can you just add an entry to the CHANGELOG under `### Added` that says something like "Added support for `fzf-lua`."

Done!

**Review by epwalsh — APPROVED** [review]
LGTM! Thank you!!

**Pull request merged**